### PR TITLE
[WIP] Credit memo order relation

### PR DESCRIPTION
**Pull request by AdamKasp**
Add a relation between order and credit memo as a first step to have easily navigation between refund/credit memo and order. 
replaced by #172

### CODE REVIEW

File: src/Resources/config/admin_routing.yml [hunk 1]
```diff
@@ -44,7 +44,6 @@ sylius_refund_credit_memo_details:
             template: "@SyliusRefundPlugin/Order/Admin/CreditMemo/details.html.twig"
             criteria:
                 id: $id
-                orderNumber: $orderNumber
```
Comment (lchrusciel) [inline src/Resources/config/admin_routing.yml]: It should stay I suppose

File: src/Resources/config/shop_routing.yml [hunk 2]
```diff
@@ -1,13 +1,13 @@
 sylius_refund_plugin_shop_order_credit_memos_partial:
-    path: /_partial/credit-memos/{orderNumber}/
+    path: /_partial/credit-memos/{orderId}/
```
Comment (lchrusciel) [inline src/Resources/config/shop_routing.yml]: I'm not sure if 
```suggestion
    path: /_partial/orders/{id}/credit-memos/
```
wouldn't be more obvious. /cc @Zales0123  @GSadee 

This path suggests IMHO only one credit memo.

File: tests/Application/config/packages/doctrine.yaml [hunk 3]
```diff
@@ -12,3 +12,6 @@ doctrine:
         charset: UTF8
 
         url: '%env(resolve:DATABASE_URL)%'
+    orm:
+        resolve_target_entities:
+            Sylius\Component\Core\Model\OrderInterface: Sylius\Component\Core\Model\Order
```
Comment (lchrusciel) [inline tests/Application/config/packages/doctrine.yaml]: Why did you add it here?

Comment (Zales0123) [reply tests/Application/config/packages/doctrine.yaml]: For some reason, it was not working without... I was doing a quick PoC so I've added it and decided to resolve it later. @AdamKasp now is "later" 😄

File: src/Resources/config/shop_routing.yml (hunk 2 above)
Comment (Zales0123) [reply src/Resources/config/shop_routing.yml]: 👍

**Pull request closed without merging**